app/assets/javascripts/application.js

```diff
@@ -25,6 +25,9 @@
 //= require controllers/ops/diagnostics_database_form_controller
 //= require controllers/ops/log_collection_form_controller
 //= require controllers/ops/tenant_form_controller
+//= require controllers/container_topology/container_topology_controller
+//= require d3/d3
```

Comment:
> Missing rails-assets-d3 from Gemfile?

actually now I noticed that d3/d3 line appears twice so I remove now this one.
As for rails-assets dependency, it supposed to be coming from rails-assets-c3